### PR TITLE
Update to .NET Standard 2.0 and to latest dependencies

### LINKED ISSUE #100: Update Xamarin.Forms dependency specification
Although the package requires the Xamarin.Forms version greater than or equal to _1.5.0.6447_ the version _2.0.0.6490_ does not work. I had to update to version _2.3.2.127_ to run.

Exception:
```
8-31 16:40:09.747	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	Probing 'java_interop_jnienv_throw'.
08-31 16:40:09.747	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	Searching for 'java_interop_jnienv_throw'.
08-31 16:40:09.747	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	Found as 'java_interop_jnienv_throw'.
08-31 16:40:09.747	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	DllImport searching in: '__Internal' ('(null)').
08-31 16:40:09.743	 5" KitKat (4.4) XXHDPI Phone	Warning	1512	dalvikvm	VFY: unable to resolve direct method 17860: Ljava/lang/Error;.<init> (Ljava/lang/String;Ljava/lang/Throwable;ZZ)V
08-31 16:40:09.743	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	Searching for 'java_interop_jnienv_new_string'.
08-31 16:40:09.743	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	dalvikvm	VFY: replacing opcode 0x70 at 0x0000
08-31 16:40:09.743	 5" KitKat (4.4) XXHDPI Phone	Info	1512	dalvikvm	Could not find method java.lang.Error.<init>, referenced from method android.runtime.JavaProxyThrowable.<init>
08-31 16:40:09.743	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	DllImport searching in: '__Internal' ('(null)').
08-31 16:40:09.743	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	Found as 'java_interop_jnienv_new_string'.
08-31 16:40:09.743	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	Probing 'java_interop_jnienv_new_string'.
08-31 16:40:09.735	 5" KitKat (4.4) XXHDPI Phone	Info	1512	MonoDroid	System.TypeLoadException: Could not resolve type with token 01000025 (from typeref, class/assembly NUnit.Runner.Services.TestOptions, nunit.runner.Droid, Version=3.6.1.0, Culture=neutral, PublicKeyToken=null)
  at Android.App.Activity.n_OnCreate_Landroid_os_Bundle_ (System.IntPtr jnienv, System.IntPtr native__this, System.IntPtr native_savedInstanceState) [0x0000f] in <d278c06ad5684d6882c743a94a93ebc2>:0 
  at (wrapper dynamic-method) System.Object:2520acca-ec1f-416d-abc3-4a879a332195 (intptr,intptr,intptr)
08-31 16:40:09.727	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	Searching for '__android_log_print'.
08-31 16:40:09.727	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	DllImport attempting to load: '/system/lib/liblog.so'.
08-31 16:40:09.727	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	Found as '__android_log_print'.
08-31 16:40:09.727	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	Probing '__android_log_print'.
08-31 16:40:09.727	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	DllImport searching in: '/system/lib/liblog.so' ('/system/lib/liblog.so').
08-31 16:40:09.727	 5" KitKat (4.4) XXHDPI Phone	Debug	1512	Mono	DllImport loaded library '/system/lib/liblog.so'.
08-31 16:40:09.727	 5" KitKat (4.4) XXHDPI Phone	Info	1512	MonoDroid	UNHANDLED EXCEPTION:
08-31 16:39:52.223	 5" KitKat (4.4) XXHDPI Phone	Error	460	WindowManager	Starting window AppWindowToken{a9439778 token=Token{a9308420 ActivityRecord{a92f5d08 u0 com.tratorimetro.library.verion.tests.android/md5b51696b2978721a0e0e6281775f51f11.MainActivity t5}}} timed out
```

### PR DESCRIPTION
**Pull request by rprouse**
Fixes #100 

This switches the main project to .NET Standard 2.0 and updates all projects to the new PackageReference format and latest dependencies.

All projects are compiling and running on all platforms. Work in progress is,

- [x] Update the cake build
- [x] Redo the nuget packaging

Major change here is that it requires the latest versions of Xamarin and also requires the Fall Creators Update and VS 2017 15.4 for UWP.

### CODE REVIEW

**Comment by rprouse** [comment]
I've got this building and I've tested all platforms now. I think this would be a great base for future work. The only potential drawback is that the NuGet package doesn't include the `txt` templates to replace the platform entry points. I never liked that and hope the templates can take care of it.

I think this is ready for review. @ChrisMaddock can you take a quick look when you have time?

**Comment by rprouse** [comment]
One other change is that UWP tests require Windows 10 Fall Creators update which is the minimum for .NET Standard 2.0. Assembly under test can still be older.

File: src/nunit.xamarin/nunit.xamarin.csproj
Comment (ChrisMaddock) [inline src/nunit.xamarin/nunit.xamarin.csproj]: Title and Description are both in here twice - I think you want the bottom pair? 🙂

File: src/runner/nunit.runner.Droid/Resources/Values/Strings.xml
Comment (ChrisMaddock) [inline src/runner/nunit.runner.Droid/Resources/Values/Strings.xml]: I added this file to fix https://github.com/nunit/nunit.xamarin/issues/57. Is it no longer needed?

File: nunit.xamarin.sln
Comment (ChrisMaddock) [inline nunit.xamarin.sln]: All these files are removed, aren't they?

File: build.cake
Comment (ChrisMaddock) [inline build.cake]: Do we not need anything platform specific at all anymore? Cool! 😎

File: src/nunit.xamarin/nunit.xamarin.csproj
Comment (ChrisMaddock) [reply src/nunit.xamarin/nunit.xamarin.csproj]: We've lost a few things vs the old nuspec here. I think we probably want some of them back in, at least.

- minclientversion
- tags
- language
- licence url
- project url
- icon url
- requirelicenceacceptance

File: build.cake
Comment (ChrisMaddock) [reply build.cake]: Should this have version passed into it somehow, so the NuGet packages are created with the 'Appveyor tag version' generated above?

File: tests/nunit.runner.tests.uwp/MainPage.xaml
Comment (ChrisMaddock) [inline tests/nunit.runner.tests.uwp/MainPage.xaml]: Templates project will also need this change. Can we make an issue when this is merged?

File: src/runner/nunit.runner.Droid/Resources/Values/Strings.xml
Comment (ChrisMaddock) [reply src/runner/nunit.runner.Droid/Resources/Values/Strings.xml]: I guess not, if we have nothing platforms specific any more?

File: nuget/uap10.0/MainPage.xaml.txt.pp
Comment (ChrisMaddock) [inline nuget/uap10.0/MainPage.xaml.txt.pp]: Getting rid of this closes https://github.com/nunit/nunit.xamarin/issues/79

**Review by ChrisMaddock — COMMENTED** [review]
Wow - this looks like it will be so much simpler!

I agree on the templates. Would be nice to see them also incorporated into dotnet new nunit at some point! 😄

File: src/nunit.xamarin/nunit.xamarin.csproj
Comment (rprouse) [reply src/nunit.xamarin/nunit.xamarin.csproj]: Will fix. Odd, I know I added all those extra fields, not sure how they got lost 😕

File: src/runner/nunit.runner.Droid/Resources/Values/Strings.xml
Comment (rprouse) [reply src/runner/nunit.runner.Droid/Resources/Values/Strings.xml]: The android test project is building and running fine with the latest and as you say, this isn't an Android library anymore 😄 

Good eye though.

File: nunit.xamarin.sln
Comment (rprouse) [reply nunit.xamarin.sln]: Missed this. Will fix.

File: build.cake
Comment (rprouse) [reply build.cake]: Probably a good idea, I will look into it.

File: nuget/uap10.0/MainPage.xaml.txt.pp
Comment (rprouse) [reply nuget/uap10.0/MainPage.xaml.txt.pp]: Yes, this is another template update too.

File: build.cake
Comment (jnm2) [reply build.cake]: I've always been curious why we have `.SetNodeReuse(false)`?

Comment (rprouse) [reply build.cake]: We have `.setNodeReuse(false)` because one of our projects failed to compile without it way back in the day so it ended up getting copied around as we started using Cake in other projects. I can try dropping it since it probably isn't needed anymore. 😄

**Comment by rprouse** [comment]
All changes based on feedback made. Version is now being injected for the AppVeyor builds and I checked the artifacts published by the AppVeyor builds.

File: src/nunit.xamarin/nunit.xamarin.csproj
Comment (ChrisMaddock) [inline src/nunit.xamarin/nunit.xamarin.csproj]: There's still duplicate Title and Description elements in here.

**Review by ChrisMaddock — CHANGES_REQUESTED** [review]
See below - the rest looks good! 😄

File: src/nunit.xamarin/nunit.xamarin.csproj
Comment (rprouse) [reply src/nunit.xamarin/nunit.xamarin.csproj]: Fixed, should be ready to merge once CI passes and you approve @ChrisMaddock

**Review by ChrisMaddock — APPROVED** [review]

**Comment by ChrisMaddock** [comment]
Woohoo! 🎉 Nice one!

**Comment by karlingen** [comment]
When can we expect a nuget release so we can upgrade to `3.10.1`? 😃

**Comment by rprouse** [comment]
@karlingen would you be interested in testing `3.10.1-dev-00197` from our AppVeyor Nuget feed https://ci.appveyor.com/nuget/nunit.xamarin and let us know how it works for you? It would help us with the release.

**Comment by karlingen** [comment]
@rprouse It runs perfectly in iOS and Android! 👍 

The only problem I have now is getting VS to run my tests as well. But that's a whole other story.

```
ERROR: System.IO.FileNotFoundException: Could not load file or assembly 'nunit.framework' or one of its dependencies
File name: 'nunit.framework'


Server stack trace: 
  at System.AppDomain.Load (System.Reflection.AssemblyName assemblyRef, System.Security.Policy.Evidence assemblySecurity) [0x001a9] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System/AppDomain.cs:777 
  at (wrapper remoting-invoke-with-check) System.AppDomain.Load(System.Reflection.AssemblyName,System.Security.Policy.Evidence)
  at System.AppDomain.Load (System.Reflection.AssemblyName assemblyRef) [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System/AppDomain.cs:714 
  at (wrapper remoting-invoke-with-check) System.AppDomain.Load(System.Reflection.AssemblyName)
  at System.Reflection.Assembly.Load (System.Reflection.AssemblyName assemblyRef) [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System.Reflection/Assembly.cs:600 
  at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName (System.Reflection.AssemblyName assemblyRef, System.Security.Policy.Evidence assemblySecurity, System.Reflection.RuntimeAssembly reqAssembly, System.Threading.StackCrawlMark& stackMark, System.Boolean throwOnFileNotFound, System.Boolean forIntrospection, System.Boolean suppressSecurityChecks) [0x0002e] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System.Reflection/MonoAssembly.cs:142 
  at System.Activator.CreateInstance (System.String assemblyString, System.String typeName, System.Boolean ignoreCase, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes, System.Security.Policy.Evidence securityInfo, System.Threading.StackCrawlMark& stackMark) [0x00049] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/activator.cs:351 
  at System.Activator.CreateInstance (System.String assemblyName, System.String typeName, System.Boolean ignoreCase, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes, System.Security.Policy.Evidence securityInfo) [0x00016] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/activator.cs:278 
  at System.AppDomain.CreateInstance (System.String assemblyName, System.String typeName, System.Boolean ignoreCase, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes, System.Security.Policy.Evidence securityAttributes) [0x0000e] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System/AppDomain.cs:391 
  at (wrapper remoting-invoke-with-check) System.AppDomain.CreateInstance(string,string,bool,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo,object[],System.Security.Policy.Evidence)
  at System.AppDomain.CreateInstanceAndUnwrap (System.String assemblyName, System.String typeName, System.Boolean ignoreCase, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes, System.Security.Policy.Evidence securityAttributes) [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System/AppDomain.cs:412 
  at (wrapper remoting-invoke-with-check) System.AppDomain.CreateInstanceAndUnwrap(string,string,bool,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo,object[],System.Security.Policy.Evidence)
  at (wrapper xdomain-dispatch) System.AppDomain.CreateInstanceAndUnwrap(object,byte[]&,byte[]&,string,string,bool)

Exception rethrown at [0]: 
  at (wrapper xdomain-invoke) System.AppDomain.CreateInstanceAndUnwrap(string,string,bool,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo,object[],System.Security.Policy.Evidence)
  at (wrapper remoting-invoke-with-check) System.AppDomain.CreateInstanceAndUnwrap(string,string,bool,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo,object[],System.Security.Policy.Evidence)
  at NUnit.Engine.Drivers.NUnit3FrameworkDriver.CreateObject (System.String typeName, System.Object[] args) [0x00000] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Drivers.NUnit3FrameworkDriver.Load (System.String testAssemblyPath, System.Collections.Generic.IDictionary`2[TKey,TValue] settings) [0x00041] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.DirectTestRunner.LoadPackage () [0x0007a] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.TestDomainRunner.LoadPackage () [0x00017] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.AbstractTestRunner.Load () [0x00000] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.MasterTestRunner.LoadPackage () [0x0001d] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.AbstractTestRunner.EnsurePackageIsLoaded () [0x00008] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.AbstractTestRunner.Run (NUnit.Engine.ITestEventListener listener, NUnit.Engine.TestFilter filter) [0x00000] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.MasterTestRunner.NUnit.Engine.ITestRunner.Run (NUnit.Engine.ITestEventListener listener, NUnit.Engine.TestFilter filter) [0x00000] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit3Runner.NUnitTestRunner.Run (NUnit.Engine.ITestEventListener listener, System.String[] nameFilter, System.String path, System.String suiteName, System.String[] supportAssemblies, System.String testRunnerType, System.String testRunnerAssembly) [0x00076] in /Users/builder/data/lanes/5965/cfe2fc05/source/monodevelop/main/src/addins/MonoDevelop.UnitTesting.NUnit/NUnit3Runner/NUnitTestRunner.cs:130 
  at NUnit3Runner.NUnitTestRunner.Run (MonoDevelop.UnitTesting.NUnit.RunRequest r) [0x00031] in /Users/builder/data/lanes/5965/cfe2fc05/source/monodevelop/main/src/addins/MonoDevelop.UnitTesting.NUnit/NUnit3Runner/NUnitTestRunner.cs:73 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System.Reflection/MonoMethod.cs:305 
System.IO.FileNotFoundException: Could not load file or assembly 'nunit.framework' or one of its dependencies
File name: 'nunit.framework'


Server stack trace: 
  at System.AppDomain.Load (System.Reflection.AssemblyName assemblyRef, System.Security.Policy.Evidence assemblySecurity) [0x001a9] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System/AppDomain.cs:777 
  at (wrapper remoting-invoke-with-check) System.AppDomain.Load(System.Reflection.AssemblyName,System.Security.Policy.Evidence)
  at System.AppDomain.Load (System.Reflection.AssemblyName assemblyRef) [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System/AppDomain.cs:714 
  at (wrapper remoting-invoke-with-check) System.AppDomain.Load(System.Reflection.AssemblyName)
  at System.Reflection.Assembly.Load (System.Reflection.AssemblyName assemblyRef) [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System.Reflection/Assembly.cs:600 
  at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName (System.Reflection.AssemblyName assemblyRef, System.Security.Policy.Evidence assemblySecurity, System.Reflection.RuntimeAssembly reqAssembly, System.Threading.StackCrawlMark& stackMark, System.Boolean throwOnFileNotFound, System.Boolean forIntrospection, System.Boolean suppressSecurityChecks) [0x0002e] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System.Reflection/MonoAssembly.cs:142 
  at System.Activator.CreateInstance (System.String assemblyString, System.String typeName, System.Boolean ignoreCase, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes, System.Security.Policy.Evidence securityInfo, System.Threading.StackCrawlMark& stackMark) [0x00049] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/activator.cs:351 
  at System.Activator.CreateInstance (System.String assemblyName, System.String typeName, System.Boolean ignoreCase, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes, System.Security.Policy.Evidence securityInfo) [0x00016] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/activator.cs:278 
  at System.AppDomain.CreateInstance (System.String assemblyName, System.String typeName, System.Boolean ignoreCase, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes, System.Security.Policy.Evidence securityAttributes) [0x0000e] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System/AppDomain.cs:391 
  at (wrapper remoting-invoke-with-check) System.AppDomain.CreateInstance(string,string,bool,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo,object[],System.Security.Policy.Evidence)
  at System.AppDomain.CreateInstanceAndUnwrap (System.String assemblyName, System.String typeName, System.Boolean ignoreCase, System.Reflection.BindingFlags bindingAttr, System.Reflection.Binder binder, System.Object[] args, System.Globalization.CultureInfo culture, System.Object[] activationAttributes, System.Security.Policy.Evidence securityAttributes) [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System/AppDomain.cs:412 
  at (wrapper remoting-invoke-with-check) System.AppDomain.CreateInstanceAndUnwrap(string,string,bool,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo,object[],System.Security.Policy.Evidence)
  at (wrapper xdomain-dispatch) System.AppDomain.CreateInstanceAndUnwrap(object,byte[]&,byte[]&,string,string,bool)

Exception rethrown at [0]: 
  at (wrapper xdomain-invoke) System.AppDomain.CreateInstanceAndUnwrap(string,string,bool,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo,object[],System.Security.Policy.Evidence)
  at (wrapper remoting-invoke-with-check) System.AppDomain.CreateInstanceAndUnwrap(string,string,bool,System.Reflection.BindingFlags,System.Reflection.Binder,object[],System.Globalization.CultureInfo,object[],System.Security.Policy.Evidence)
  at NUnit.Engine.Drivers.NUnit3FrameworkDriver.CreateObject (System.String typeName, System.Object[] args) [0x00000] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Drivers.NUnit3FrameworkDriver.Load (System.String testAssemblyPath, System.Collections.Generic.IDictionary`2[TKey,TValue] settings) [0x00041] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.DirectTestRunner.LoadPackage () [0x0007a] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.TestDomainRunner.LoadPackage () [0x00017] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.AbstractTestRunner.Load () [0x00000] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.MasterTestRunner.LoadPackage () [0x0001d] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.AbstractTestRunner.EnsurePackageIsLoaded () [0x00008] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.AbstractTestRunner.Run (NUnit.Engine.ITestEventListener listener, NUnit.Engine.TestFilter filter) [0x00000] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit.Engine.Runners.MasterTestRunner.NUnit.Engine.ITestRunner.Run (NUnit.Engine.ITestEventListener listener, NUnit.Engine.TestFilter filter) [0x00000] in <7934713e6a5945a48b191c3250979bc9>:0 
  at NUnit3Runner.NUnitTestRunner.Run (NUnit.Engine.ITestEventListener listener, System.String[] nameFilter, System.String path, System.String suiteName, System.String[] supportAssemblies, System.String testRunnerType, System.String testRunnerAssembly) [0x00076] in /Users/builder/data/lanes/5965/cfe2fc05/source/monodevelop/main/src/addins/MonoDevelop.UnitTesting.NUnit/NUnit3Runner/NUnitTestRunner.cs:130 
  at NUnit3Runner.NUnitTestRunner.Run (MonoDevelop.UnitTesting.NUnit.RunRequest r) [0x00031] in /Users/builder/data/lanes/5965/cfe2fc05/source/monodevelop/main/src/addins/MonoDevelop.UnitTesting.NUnit/NUnit3Runner/NUnitTestRunner.cs:73 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in /Users/builder/jenkins/workspace/build-package-osx-mono/2017-12/external/bockbuild/builds/mono-x64/mcs/class/corlib/System.Reflection/MonoMethod.cs:305 
```

**Comment by WonyoungChoi** [comment]
@rprouse It runs well in Tizen also. 😃👍   I really hope that the new version of nunit.xamarin will be released. Do you have any plans for the release?

**Comment by rprouse** [comment]
@WonyoungChoi I was hoping to do a major rewrite of this using the NUnit Engine. I don't think any of the team is currently doing Xamarin development, so this has fallen by the wayside a bit. Sorry. If I get a chance, I'll see if I can pull this together for a release.

**Pull request merged**
